src/pgwire/src/protocol.rs

```diff
@@ -1390,7 +1456,7 @@ where
                 }
                 command_complete!()
             }
-            ExecuteResponse::Subscribing { rx } => {
+            ExecuteResponse::Subscribing { rx, ctx_extra } => {
```

Comment:
Maybe not for this PR: having `SUBSCRIBE` only log on completion might be bad because in the happy path, it won't complete. Can we make it special so that we log on start so we get full `SUBSCRIBE` text even if a query didn't complete?

I alllmost want the same thing for slow path peeks, because those have a higher potential to crash loop a clusterd, and in those cases we won't have the crash-inducing query.

Reply:
We will get the sql text in those cases as we log that on start! The main things we log on end are the status and the time to completion. So if a subscribe or slow path select is still running we’ll just show null for those.